src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestedCashAdvanceServiceImpl.java

```diff
@@ -27,7 +27,7 @@ public boolean isDuplicateConcurRequestCashAdvance(ConcurRequestedCashAdvance co
         Map<String, String> fieldValues = new HashMap<String, String>();
         
         fieldValues.put(ConcurPropertyConstants.ConcurRequestedCashAdvance.EMPLID, concurRequestedCashAdvance.getEmployeeId());
-        fieldValues.put(ConcurPropertyConstants.ConcurRequestedCashAdvance.REQUEST_ID, concurRequestedCashAdvance.getRequestId());
+        fieldValues.put(ConcurPropertyConstants.ConcurRequestedCashAdvance.CASH_ADV_KEY, concurRequestedCashAdvance.getCashAdvanceKey());
```

Comment:
Just to confirm, do you know if the mapping between request IDs and cash advance keys is 1-1? Or is it 1-N instead? If the latter, then perhaps additional criteria are needed to help determine uniqueness.

Reply:
The Concur rep who spoke with Tami and I indicated that the requestID and cashAdvanceKey were 1-1. Looking at our database table, I also saw that....which allowed for the secondary composite key change.

Reply:
Ok, sounds good. Thanks for checking!